lib/fbe/github/graphql/client.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# MIT License
+#
+# Copyright (c) 2024 Zerocracy
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+require 'graphql/client'
+
+# The GitHub GraphQL client
+class Fbe::GitHub::GraphQL::Client
+  def initialize(token:)
+    http = Fbe::GitHub::GraphQL::HTTP.new(token)
+    schema = GraphQL::Client.load_schema(http)
+    @client = GraphQL::Client.new(schema:, execute: http)
+    @client.allow_dynamic_queries = true
+  end
+
+  def query(query_string)
+    query  = @client.parse(query_string)
+    result = @client.query(query)
```

Comment:
The same [redundant variable](https://www.yegor256.com/2015/09/01/redundant-variables-are-evil.html):
```ruby
result = @client.query(@client.parse(query_string))
```